integration/setup/setup.go

```diff
@@ -60,27 +60,27 @@ func HostPeerVPC(c *client.AWS, f *framework.Framework) error {
 	return nil
 }
 
-func Resources(c *client.AWS, f *framework.Framework) error {
+func Resources(c *client.AWS, g *framework.Guest, h *framework.Host) error {
 	var err error
 
 	{
 		// TODO configure chart values like for the other operators below.
-		err = f.InstallCertOperator()
+		err = h.InstallCertOperator()
 		if err != nil {
 			return microerror.Mask(err)
 		}
-		err = f.InstallNodeOperator(template.NodeOperatorChartValues)
+		err = h.InstallNodeOperator(template.NodeOperatorChartValues)
 		if err != nil {
 			return microerror.Mask(err)
 		}
-		err = f.InstallAwsOperator(template.AWSOperatorChartValues)
+		err = h.InstallAwsOperator(template.AWSOperatorChartValues)
 		if err != nil {
 			return microerror.Mask(err)
 		}
 	}
 
 	{
-		err = f.InstallCertResource()
+		err = h.InstallCertResource()
```

Comment:
This chain of Install calls looks like there is a lot to know about framework outside it. WDYT about creating `h.Install()` which would do all of that?

Maybe it can be also part of a `Setup()`.

Reply:
There is definitely a lot to be refactored still but we have to do one step at a time and I already did a lot. I actually would like to make some progress on adding new integration tests now.

Reply:
Just saying. This is cool already.

Reply:
Yeah yeah, You are right.